### DISCUSSION TITLE
Who else think that the 'cat' blocks are SOO cute!?

### DISCUSSION THREAD

**Discussion by Mr-Programmer-777** [ScratchAddons/ScratchAddons#1792] [Show and tell]
I love these guys!
![CAT BLOCK!!](https://user-images.githubusercontent.com/77010833/110666902-a0b7bf80-81c1-11eb-91fd-d9dadf23d72a.png)

Yours cat-lovingly,
MrProgrammer777

**Comment by TheColaber** [comment]
This is not the place to discuss whether things are cute or not.

**Reply by mxmou** [reply]
I changed the category, but as you said this doesn't really belong here.

**Reply by Mr-Programmer-777** [reply]
Just wanted to thank you for bringing them back!
